modules/mono/glue/GodotSharp/GodotSharp/Core/NativeInterop/NativeFuncs.cs

```diff
@@ -153,9 +153,9 @@ public static partial void
         internal static partial void godotsharp_callable_call_deferred(in godot_callable p_callable,
             godot_variant** p_args, int p_arg_count);
 
-        // GDNative functions
+        // GDExtension functions
 
-        // gdnative.h
+        // gdextension.h
```

Comment:
This doesn't make much sense, as this mimics functions from the old the old `gdnarive.h` header. Also, AFAIK `gdextension.h` doesn't exist either.

Reply:
Hmm, well, I renamed `native_extension.h` to `gdextension.h` now, so that file exists. But indeed it does not correspond to the old `gdnative.h` file. So I agree on the change, I'll fix it.